inference/object_completion.py

```diff
@@ -37,6 +37,8 @@
 
 recon, skip = model.forward(data=voxels)
 
+recon = torch.exp(recon) - 1.0
```

Comment:
```suggestion
recon = recon
```
Why are we doing this? The output of the network is supposed. This would make sense to me if we directly inside the network applied the log and +1. However we do that outside the network in the training loop. So this doesn't make too much sense to me

Reply:
I just follow what E3 they do to 3depn network. I think the output of 3depn network is in log space